### PR TITLE
add optional answer feedback strings to match and multi

### PR DESCRIPTION
**Pull request by cpirich**
- add optional answer feedback strings to match and multi
- show them in gamelab predictive level feedback dialog
- clean up some style issues from the contained_levels PR

### CODE REVIEW

**Comment by cpirich** [comment]
@breville

File: code-studio/src/js/levels/multi.js [hunk 1]
```diff
@@ -220,6 +223,21 @@ Multi.prototype.getResult = function (dontAllowSubmit) {
   };
 };
 
+// called by external code that will display answer feedback
+Multi.prototype.getCurrentAnswerFeedback = function () {
+  if (!this.answersFeedback) {
+    return;
```
Comment (breville) [inline code-studio/src/js/levels/multi.js:229]: Curious but unsure: should we explicitly return null here?

**Comment by breville** [comment]
This approach looks sound to me, given that you want it for both Match & Multi going forward.  That said, it'd be good for @mehalshah @joshlory and maybe @wjordan to take a look too since they are all interested in DSL defined levels.

File: code-studio/src/js/levels/multi.js (hunk 1 above)
Comment (cpirich) [inline code-studio/src/js/levels/multi.js:229]: I think `undefined` is quite reasonable. Calling code will treat them the same way. The lack of a return value is sort of accurate "I have no answer feedback". I find that I don't have to use `null` nearly as much in JavaScript as in C/C++

**Comment by cpirich** [comment]
Happy to adjust based on feedback if anyone has feedback beyond Brendan's, but since I haven't heard anything yet today, I'm going to go ahead and merge before `staging-next` goes away...

**Pull request merged**